src/operators/suffix.ts

```diff
@@ -0,0 +1,172 @@
+import { OperatorValidator, OperatorOptions, Operator } from '../operator';
+
+/**
+ * A SuffixedObject is an object that has had FS types appended to properties.
```

Comment:
Since this is a general purpose operator, maybe we can leave out the FS-specific comments?

Reply:
As I re-read it this morning I thought the same.  Technically suffixing extends beyond our needs.